mllib/src/main/scala/org/apache/spark/mllib/clustering/LocalKMeans.scala

```diff
@@ -46,17 +46,20 @@ private[mllib] object LocalKMeans extends Logging {
 
     // Initialize centers by sampling using the k-means++ procedure.
     centers(0) = pickWeighted(rand, points, weights).toDense
+    val costArray = points.map { point =>
```

Comment:
can be `.map(KMeans.fastSquaredDistance(_, centers(0)))` ? I forget whether this is actually allowed as syntax